### PR TITLE
Moved eff to projects and added myself in the list of authors

### PR DESCRIPTION
**Pull request by etorreborre**
@larsrh please review.

### CODE REVIEW

File: _data/authors.yml [hunk 1]
```diff
@@ -246,3 +246,8 @@ data_fly:
   full_name: "Zhenhao Li"
   portrait: "https://avatars0.githubusercontent.com/u/10957195?v=3&s=80"
   bio: "Zhenhao Li is a data engineer and data scientist at Connecterra, a data science and IoT startup. He is responsible for making scalable data processing jobs and pipelines and making sure data science insights are generated and delivered in real time. Before joining Connecterra, Zhenhao worked for Accenture in the area of big data and IoT technology consulting, helping major clients to adapt new technologies such as Kafka and Flink, the immutable data paradigm, and functional programming to gain business value faster. He holds a bachelor degree in software engineering and a master degree in logic. He was doing a PhD in mathematical logic at the University of Amsterdam when decided to change his career path to big data and data science. He loves functional programming, and Scala is his primary language for engineering work."
+etorreborre:
```
Comment (larsrh) [inline _data/authors.yml]: This is not necessary. We use this currently for blog posts & talks at events. Unless you want to write a blog post about `eff`, of course :wink:

Comment (adelbertc) [inline _data/authors.yml]: So what you're trying to say is... @etorreborre needs to write a blog post about Eff right? :D

Comment (larsrh) [inline _data/authors.yml]: Let's say "is encouraged to" :smile:

**Comment by etorreborre** [comment]
Ok, I removed myself from the authors file. I'll keep that in mind when I write my first blog post :-).

File: _projects/eff.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+---
+layout: post
+title: "eff"
+category: "effects"
+description: "Extensible effects are an alternative to monad transformers for computing with effects in a functional way. This library is based on the “free-er” monad and an “open union” of effects described by Oleg Kiselyov in “Freer monads, more extensible effects“"
```
Comment (adelbertc) [inline _projects/eff.md]: Seems there's unicode quotes around the paper title, and the closing quote is in the wrong direction?

Comment (etorreborre) [inline _projects/eff.md]: I fixed the closing quote. Do you think it is better to use ASCII quotes? And do you know if they are escaped with `\` in that case?

**Pull request merged**